cpp/ps0001/problem 2.cpp

```diff
@@ -1,11 +1,17 @@
-#include<iostream>
+#include <iostream>
 using namespace std;
+
 int main()
 {
-int number1,number2,number3;
-cout << "Please Enter 3 numbers \n" << endl;
-cin >> number1>>number2>>number3;
-int sum=(number1+number2+number3);
- cout <<" sum equal\n "<<sum<<" \n";
-return 0;
-}
\ No newline at end of file
+  // prompting user for 3 numbers to add 1/2
+  unsigned short number1, number2, number3;
+  cout << "Please Enter 3 numbers \n" << endl;
+  cin >> number1 >> number2 >> number3;
```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add